docs/release-notes/remove-old-tempalte-apis.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Breaking Change**
+
+-  Old template CRUD endpoints have been removed. These used to be serviced under `/templates/*`.
+   Please use the new apis located at `/api/v1/templates/*` instead.
```

Comment:
The previous template CRUD endpoints have been removed from the /templates/* location.
The new APIs can be found at /api/v1/templates/*.

Reply:
thanks, that sounds good. The second sentence could be misinterpreted I think. maybe we should remove "new" from that? 
I want to make sure the reader won't think that the new apis are added here and that they did exist before and people should use those if it affetcs them